src/main/java/uk/gov/hmcts/reform/sendletter/entity/LetterRepository.java

```diff
@@ -2,18 +2,32 @@
 
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Lock;
+import org.springframework.data.jpa.repository.Query;
 
+import java.sql.Timestamp;
 import java.util.Optional;
 import java.util.UUID;
 import java.util.stream.Stream;
 import javax.persistence.LockModeType;
 
 public interface LetterRepository extends JpaRepository<Letter, UUID> {
+
+    String STALE_LETTERS_QUERY = "select"
```

Comment:
could be better if we could store in some external file similar to https://github.com/spring-projects/spring-data-jpa/blob/master/src/test/resources/META-INF/jpa-named-queries.properties